File: FTP Client for WinForms using Ultimate FTP library/[VB]-FTP Client for WinForms using Ultimate FTP library/VB/Ftp/FtpLoginInfo.vb
Imports System.Text

Namespace ClientSample.Ftp
	Friend Class FtpLoginInfo
		Public Const PasvMode As String = "PasvMode"
		Public Const SecurityMode As String = "SecMode"
		Public Const Certificate As String = "Cert"
		Public Const ClearCommandChannel As String = "ccc"

		Public Shared Sub SaveConfig(ByVal settings As SettingInfoBase)
			settings.SaveProperty(PasvMode)
			settings.SaveProperty(Certificate)
			settings.SaveProperty(SecurityMode)
			settings.SaveProperty(ClearCommandChannel)
		End Sub

		Public Shared Sub LoadConfig(ByVal settings As SettingInfoBase)
			settings.LoadProperty(PasvMode, True)
			settings.LoadProperty(Certificate, String.Empty)
			settings.LoadProperty(SecurityMode, 0)
			settings.LoadProperty(ClearCommandChannel, False)
		End Sub
	End Class
End Namespace

File: FTP Client for WinForms using Ultimate FTP library/[C#]-FTP Client for WinForms using Ultimate FTP library/C#/Ftp/FtpLoginInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ClientSample.Ftp
{
    class FtpLoginInfo
    {
        public const string PasvMode = "PasvMode";
        public const string SecurityMode = "SecMode";
        public const string Certificate = "Cert";
        public const string ClearCommandChannel = "ccc";

        public static void SaveConfig(SettingInfoBase settings)
        {
            settings.SaveProperty(PasvMode);
            settings.SaveProperty(Certificate);
            settings.SaveProperty(SecurityMode);
            settings.SaveProperty(ClearCommandChannel);
        }

        public static void LoadConfig(SettingInfoBase settings)
        {
            settings.LoadProperty(PasvMode, true);
            settings.LoadProperty(Certificate, string.Empty);
            settings.LoadProperty(SecurityMode, 0);
            settings.LoadProperty(ClearCommandChannel, false);
        }
    }
}
